_includes/corridors/bwp-bank.md

```diff
@@ -0,0 +1,181 @@
+Barclays Bank, Jwaneng, 291067
+Barclays Bank, Kanye, 291267
+Barclays Bank, Kasane, 292067
+Barclays Bank, Letlhakane, 293967
+Barclays Bank, Lobatse, 290867
+Barclays Bank, Mahalapye, 291367
+Barclays Bank, Masunga, 292267
+Barclays Bank, Maun, 291967
+Barclays Bank, Merafhe/Gemvas, 290567
+Barclays Bank, Mochudi, 292467
+Barclays Bank, Mogoditshane, 292967
+Barclays Bank, Molepolole, 292567
+Barclays Bank, Moshupa, 292867
+Barclays Bank, Orapa, 290967
+Barclays Bank, Palapye, 291567
+Barclays Bank, Phakalane, 293667
+Barclays Bank, Ramotswa, 292767
+Barclays Bank, Serowe, 291467
+Barclays Bank, Shakawe, 293367
+Barclays Bank, Sowa, 293067
+Barclays Bank, Tlokweng, 292667
+Barclays Bank, Tsabong, 292167
+Barclays Bank, Tutume, 291867
+Capital Bank, Gaborone Main Branch, 800267
+Capital Bank, Head Office, 800167
+Capital Bank, Broadhurst, 800467
+Capital Bank, Francistown, 800367
+Capital Bank, Mogoditshane, 800567
+First National Bank, Airport Junction, 288267
+First National Bank, Code Line Clearing, 284367
+First National Bank, Electronic Banking, 283767
+First National Bank, First Card, 282767
+First National Bank, First Funding Motor, 284767
+First National Bank, Gaborone Industrial, 281667
+First National Bank, Gaborone Main, 281467
+First National Bank, Gaborone Mall, 282867
+First National Bank, Head Office, 282067
+First National Bank, Kgale View, 284567
+First National Bank, Private Clients, 285467
+First National Bank, Property Division, 281067
+First National Bank, Rail Park Mall, 288067
+First National Bank, Selebi Phikwe, 285067
+First National Bank, Wes Bank, 281567
+First National Bank, WesBank Francistown, 283467
+First National Bank, WesBank Motor, 283367
+First National Bank, West Bank Corporate, 283267
+First National Bank, Broadhurst, 281267
+First National Bank, Corporate, 282267
+First National Bank, Francistown, 281867
+First National Bank, Ghanzi, 286067
+First National Bank, Jwaneng, 283067
+First National Bank, Kanye, 281967
+First National Bank, Kasane, 285167
+First National Bank, Letlhakane, 285567
+First National Bank, Lobatse, 281767
+First National Bank, Mahalapye, 282467
+First National Bank, Maun, 282367
+First National Bank, Molepolole, 285667
+First National Bank, Palapye, 283167
+First National Bank, RMB, 287867
+First National Bank, Rverwalk, 285267
+First National Bank, Serowe, 285367
+First National Bank, Treasury, 282167
+Stanbic Bank, CBD Square, 65167
+Stanbic Bank, CPS Credit Card, 65667
+Stanbic Bank, Credit card, 65567
+Stanbic Bank, Fair grounds, 64967
+Stanbic Bank, Gaborone Industrial, 61967
+Stanbic Bank, Gaborone West, 60167
+Stanbic Bank, Head Office, 60467
+Stanbic Bank, Selebi Phikwe, 65767
+Stanbic Bank, Work Place Banking, 65867
+Stanbic Bank, Broadhurst, 60367
+Stanbic Bank, Francistown, 64067
+Stanbic Bank, Kgale, 60267
+Stanbic Bank, Maun, 64767
+Stanbic Bank, Mogoditshane, 65367
+Stanbic Bank, Molepolole, 65467
+Stanbic Bank, Palapye, 65067
+Stanbic Bank, VAF, 60567
```

Comment:
Looking at the codes you shared [here](https://www.sci.co.bw/banking-in-botswana) it seems all Stanbic Bank codes should start with a 0 (also the fields doc specifies we should send exactly 6 digits)... I believe the zeroes were trimmed by mistake in the CSV...